### PR TITLE
Registry

### PR DESCRIPTION
**Pull request by fernandosanchezmunoz**
Docker registry package. Tested and verified.

### CODE REVIEW

**Comment by ssk2** [comment]
Nice!!!

File: repo/packages/R/registry/0/config.json [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+					"type": "string",
+					"default": "registry"
+				}
+			}
+		},
+		"registry": {
+			"type": "object",
+			"description": "registry service configuration properties",
+			"properties": {
+				"cpus": {
```
Comment (ryadav88) [inline repo/packages/R/registry/0/config.json]: move `cpu` and `memory` to `service` object above

File: repo/packages/R/registry/0/config.json [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+							"type": "string"
+						},						
+						"password": {
+							"description": "Password to use in the remote proxy.",
+							"type": "string"
+						}
+					}
+				}
+			},
+			"required": [
```
Comment (ryadav88) [inline repo/packages/R/registry/0/config.json]: move to `service`

File: repo/packages/R/registry/0/config.json [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+					"default": 0.2,
+					"minimum": 0.2
+				},
+				"mem": {
+					"description":  "Memory to allocate to each service instance.",
+					"type": "number",
+					"default": 256.0,
+					"minimum": 256.0
+				},
+				"search_backend": {
```
Comment (ryadav88) [inline repo/packages/R/registry/0/config.json]: what will this be used for? Are there any other supported backends apart from `sqlalchemy`?
should we add this in dcos/examples?

File: repo/packages/R/registry/0/marathon.json.mustache [hunk 4]
```diff
@@ -0,0 +1,84 @@
+{
+  "id": "{{service.name}}",
+  "cpus": {{registry.cpus}},
+  "mem": {{registry.mem}},
```
Comment (ryadav88) [inline repo/packages/R/registry/0/marathon.json.mustache]: change to service.cpus and service.mem after making changes in config.json

File: repo/packages/R/registry/0/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name": "registry",
+  "version": "2.5.1-0.1",
+  "scm": "https://github.com/registry/registry-server.git",
```
Comment (fernandosanchezmunoz) [inline repo/packages/R/registry/0/package.json]: what exactly doesn't look correct? need 1.7 for persistent storage, registry version is 2.5.1, packaging version is 0.1

File: repo/packages/R/registry/0/config.json (hunk 2 above)
Comment (fernandosanchezmunoz) [inline repo/packages/R/registry/0/config.json]: why?

File: repo/packages/R/registry/0/package.json (hunk 5 above)
Comment (ryadav88) [inline repo/packages/R/registry/0/package.json]: https://github.com/registry/registry-server.git

Comment (fernandosanchezmunoz) [inline repo/packages/R/registry/0/package.json]: right thanks -- corrected.

**Pull request merged**